### ISSUE TITLE
Convenience dropdown for custom tokens

### ISSUE THREAD

**Issue by Soleone** [EOSEssentials/Scattellet#2]
### Problem

Custom token support is great. But in order to see their balance it still requires users to know the eos account names of the project and the exact name of the token symbol.

### Suggestion

It would be great if the popular tokens would automatically show up for users in a simple dropdown, similar to what eospark.com is doing:

![image](https://user-images.githubusercontent.com/2923/41878948-53f0ad6a-78a5-11e8-9aab-fc97bdf6877e.png)

Basically an endorsed map of `account name` --> `token symbol` map. And then users can add their own entries like they can do now (either still only temporarily or maybe saved in local storage for convenience, but don't think that's necessary until there's way more tokens).

Random ~shower~ github-issue-writing thought: storing preferences of the app in the blockchain could be nice for convenience.

**Comment by kesar** [comment]
design from Only Koreos suggestion:

<img width="901" alt="screen shot 2018-06-27 at 13 07 50" src="https://user-images.githubusercontent.com/1288106/41970418-2489d7ce-7a0b-11e8-99ab-dd0b24661e8f.png">

**Comment by Kersten-CR** [comment]
Once a token is selected it would be great to see some further contract information such as full name and contract ID. Since token shortcuts are not harmonized this will give further security that wrong tokens are not sent to wrong address.
